mobile/src/src/components/Menu/MenuPage.jsx

```diff
@@ -18,8 +18,8 @@ class MenuPage extends Component {
 
   componentDidUpdate = async (_prevProps, prevState) => {
     if (prevState != this.state) {
-      const response = await ProfileEndpoint.get();
-      this.setState({ user: response.data });
+      const userData = await getUserData();
```

Comment:
No I was saying that after you got the response, save the new user information to async storage with setUserData(response.data);

Reply:
I did save the new user info to async storage with setUserData(response.data) in EditComponent.jsx line 70. I used componentDidUpdate to update the stuff in the render() after the state changes or else it doesn't update in the render even though it actually updated

Reply:
Oops my bad! That's fantastic!